public/javascript/pump/view.js

```diff
@@ -630,6 +631,10 @@
                 streams.notifications = view.minorStreamView.model;
             }
             return streams;
+        },
+        closeDropdown: function(e) {
+            e.preventDefault();
```

Comment:
Is this really needed? I guess this is to prevent the browser from doing a full page refresh but it seems like that _should_ already be handled by view code